packages/gatsby-source-wordpress/package.json

```diff
@@ -8,6 +8,13 @@
     "lodash": "^4.17.4",
     "qs": "^6.4.0"
   },
+  "devDependencies": {
```

Comment:
Could you remove all the babel stuff here? We maintain a common babel setup at the base of the repo. If it's not adequate then please suggest/make modifications there.